app/code/community/Robm/Tracking/sql/robm_tracking_setup/mysql4-install-0.0.1.php

```diff
@@ -6,5 +6,5 @@
 
 $installer = $this;
 $installer->startSetup();
-$installer->run("ALTER TABLE {$this->getTable('sales_flat_order')} ADD `order_campaign_params` VARCHAR(255) NULL;");
+$installer->run("ALTER TABLE {$this->getTable('sales_flat_order')} ADD `order_campaign_params` TEXT NULL;");
```

Comment:
Shouldn't that be another migration/update version? Otherwise users who already installed it can't update properly.

Reply:
Sure Rob. I wasn't sure if you still mantain it..
I'll update it soon.
Let me ask you something.. I saw you had this part of code in Observer.php:

```php
$db = Mage::getSingleton('core/resource')->getConnection('core_write');
try {
    $db->insertOnDuplicate($db->getTableName('robm_tracking'), array('date' => date('Y-m-d'), 'utm_source' => $utmSource, 'clicks' => 1), array('clicks' => new Zend_Db_Expr('`clicks` +1')));
            } catch (Exception $e) {
                Mage::logException($e);
            }
```

But I didn't see the table creation or any reference to this table in config.xml. I removed it.
Would you mind to add it too in a new update script? (perhaps the same script we are going to update)
Or just paste here the script/ddl and I add it to the updated PR.